examples/cog-blob.js

```diff
@@ -0,0 +1,25 @@
+import GeoTIFF from '../src/ol/source/GeoTIFF.js';
+import Map from '../src/ol/Map.js';
+import TileLayer from '../src/ol/layer/WebGLTile.js';
+
+fetch('data/example.tif')
+  .then((response) => response.blob())
+  .then((blob) => {
+    const source = new GeoTIFF({
+      sources: [
+        {
+          blob: blob,
+        },
+      ],
+    });
+
+    const map = new Map({
+      target: 'map',
+      layers: [
+        new TileLayer({
+          source: source,
+        }),
+      ],
+      view: source.getView(),
```

Comment:
Looks like the example is suffering from #10508 because an extent is set and only one resolution is configured.
```suggestion
      view: source.getView().then(function (viewConfig) {
        viewConfig.showFullExtent = true;
        return viewConfig;
      }),
```
Alternatively you could remove the `resolutions` config:
```patch
-      view: source.getView(),
+      view: source.getView().then(function (viewConfig) {
+        delete viewConfig.resolutions;
+        return viewConfig;
+      }),
```

Reply:
Yeah, I was wondering where the flickering comes from. Thanks for the hint, will fix the example tomorrow.

Reply:
Fixed the example.

Reply:
I haven't tried, but assume we could instead just add more resolutions.

```js
source.getView().then(config => (
  {
    ...config
    resolutions: new Array(4).fill(0).map((v, i) => config.resolutions[0] / Math.pow(2, i))
  }
))
```

Or similar